Doc/tutorial/introduction.rst

```diff
@@ -189,6 +189,29 @@ the first quote::
    >>> print(r'C:\some\name')  # note the r before the quote
    C:\some\name
 
+There is one subtle aspect to raw strings: a raw string may not end in an odd
```

Comment:
The text of the addition is fine. But I am wondering whether such a subtle point should be part of the `introduction.rst`. Perhaps keep only the first line and refer to a location with details?

(not sure what would be a better location)

Reply:
It's not strictly a part of this file, see https://docs.python.org/3/reference/lexical_analysis.html:

`... even a raw string cannot end in an odd number of backslashes ...`

Reply:
The text you added in this PR will end up here: https://docs.python.org/3/tutorial/introduction.html#strings right?

It is an informal introduction, so I would write something like:

There is one subtle aspect to raw strings: a raw string may not end in an odd
number of ``\`` characters. For details see [String and Bytes literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals)

But the text added is correct, so it is mostly a matter of style.

Reply:
Re-reading this, I realize that my comment above doesn't actually address your original concern on whether this is too subtle of a point for the tutorial introduction - IMO, since the section above does talk about escaping quotes as well as raw strings, I think it's somewhat natural to at least mention the case with them combined.

As for how brief a mention, the current phrasing can definitely be made more brief. I think also that just one of the workarounds can be left in, while the other two examples put elsewhere, along with the wording specific to Windows paths. I don't think the reference is the most appropriate place, maybe instead in some FAQ entry or a howto page?

Reply:
An FAQ entry sounds reasonable. Maybe we do something like `There is one subtle aspect to raw strings:a raw string may not end in an odd number of \ characters; see [the FAQ entry] for workarounds`?